docs/Field-Mapping.md

```diff
@@ -283,22 +283,30 @@ All payment methods:
     * `clearing_bankbic` ->  `paidFromBIC` 
    
 `INVOICE-KLARNA`:
-  * _Required_ `language` -> custom field `languageTag` of Type String on the CT Payment
-    * (may be moved to the Order, Cart and Customer Objects later)
+  * _Required_ 
+    * `language` -> custom field `languageTag` of Type String on the CT Payment
```

Comment:
`language` is Payone request field, `languageTag` is legacy custom field in CTP project (guess descended from some old projects).